AmazonChimeSDKDemo/AmazonChimeSDKDemo/RosterModel.swift

```diff
@@ -22,6 +22,9 @@ class RosterModel: NSObject {
 
     static func convertAttendeeName(from info: AttendeeInfo) -> String {
         let externalUserIdArray = info.externalUserId.components(separatedBy: "#")
+        if externalUserIdArray.isEmpty {
```

Comment:
just curious, does transcription returns nil or empty string on externalUserId? In Android, I have seen something like if (externUserId == null || attendeeId == null || isEmpty....)

Reply:
It returns an empty string since Protobuf uses default/zero value for the type if a field is missing (for string it's `""`) during unmarshal. It was causing a crash (WTBugs-25710) when we are trying to access `externalUserIdArray[1]` directly without checking (if it were `nil` it would crash at the line above).